Improve debugging in visual studio code (#241)
Deltares/imod_coupler: README.md

- The tests can then be run with:

```sh
pixi run tests
```

- Lint the codebase with:

```sh
pixi run lint
```

- When developing with visual studio code, it is recommended to open the application via `open-vscode.bat`.
  This will open the application in a new vscode window with the correct environment variables set.

### Debugging

When debugging the unit tests in visual studio code with the test explorer, you can encounter some problems.

> MODFLOW is not so keen on multi-threading, and when debugging a test, make sure that you only have one test running at a time.
> 

Reviewer: ```suggestion
Both MODFLOW 6 and MetaSWAP might behave unpredicateble when being initialized and finalized multiple times.
When debugging a test, make therefore sure that you only have one test running at a time.
```

> When running the unit tests, this is not the case, since there is a switch statement that determines if we should call subprocess.run(), or stay within the main thread.

Reviewer: ```suggestion
When you only run, not debug, the tests, this is not the case, since there is a switch statement that determines if we should call `subprocess.run()`, or stay within the main thread.
```

See the fixture for `imodc_executable` for more information.

### Troubleshooting

If you encounter errors while running the tests, it might be that your pip dependencies are outdated.
This happens when you have pulled the latest changes from imod_coupler.
In that case you need to update the pip dependencies as well.
Try running:

```sh

> pixi run install

Reviewer: ```suggestion
pixi run update-git-dependencies
```
